patches/chrome-browser-about_flags.cc.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
-index 62f4b41a2ac4..4f63fa202bca 100644
+index 62f4b41a2ac4cb1d5da3dd374492a670c62c8812..4f63fa202bca93014fe7cc5b8ba4f6fe54b6493f 100644
```

Comment:
This (and several other patches) in this PR appear to have been accidentally generated by `npm run update_patches` are should be removed because they are unrelated to the functionality of this PR. A separate PR should be created to resolve this superfluous patch generation issue. See https://bravesoftware.slack.com/archives/C7VLGSR55/p1542747621032500 for further discussion/context.

Reply:
removed these 👍